parsers/evfoldparser.py

```diff
@@ -0,0 +1,24 @@
+from utils.exceptions import InvalidFormat
+from operator import itemgetter
+
+def EvfoldParser(input):
+    contents = input.split('\n')
+    output = []
+
+    for line in contents:
+        line = line.lstrip().split()
+        if not line or line[0].isalpha():
+            continue
+        elif line[0].isdigit() and line[2].isdigit() and len(line) >= 5:
+            if abs(int(line[0]) - int(line[2])) >= 5:
+                output.append((int(line[0]), int(line[2]), float(line[5])))
+
+    if not output:
+        raise InvalidFormat('Unable to parse contacts')
+    else:
+        output = sorted(output, key=itemgetter(2), reverse=True)
+        return tuple(output)
+
+test=open('/Users/shahrammesdaghi/Downloads/w9dy28.evfold','r')
+text=test.read()
+print(EvfoldParser(text))
```

Comment:
This lines are executed every time you import EvfoldParser, and it breaks the code because that file is obviously not there... It is ok to use this while testing on your own computer, but make sure to remove any reference to local files before you commit and push!

Reply:
Also add an empty new line at the end of the file (PEP-8 convention)